yudien/data/udn_test_cases/strconcat/input.json

```diff
@@ -0,0 +1,4 @@
+{
+    "func": "__concat",
```

Comment:
Is this only going to be single function testing?  If yes, thats good.

If we want to use the same code to do integration testing, and I dont see why we wouldnt, we should change this to "statement" instead of "func", as a Function is 1 thing, but a statement is a series of them.  A full UDN string (in 1 string/line)

Reply:
Done